
#goal
int32 seconds
# nav_msgs/Odometry od
---
#result
sensor_msgs/Image[] allPictures #an array of pictures
---
#feeback
sensor_msgs/Image lastPic